pcdsdevices/sim/motor.py

```diff
@@ -0,0 +1,69 @@
+import time
+import threading
+
+from ophyd.device import Device, Component as Cpt
+from ophyd.positioner import SoftPositioner
+from ophyd.signal import AttributeSignal
+from ophyd.sim import SynAxis
+
+from pcdsdevices.mv_interface import FltMvInterface
+
+
+class SynAxisInterface(FltMvInterface, SynAxis):
+    """
+    SynAxis that has our special interface on it.
+
+    This can be used to test when you need the readback_func feature or if you
+    just want your test motor to finish immediately.
+    """
+    def move(self, position, *args, **kwargs):
+        return super().set(position)
+
+
+class SimMotor(FltMvInterface, SoftPositioner, Device):
```

Comment:
If we were doing everything from the start I'd say `Sim` is better, but might vote for keeping the `Syn-` prefix.

Reply:
See above about potential renames. I also don't *really* like these names.

Reply:
This one should not be named `Syn` though so that nobody expects it to have all the features of a `SynAxis`.